### PR TITLE
Update Network Capture with new system-wide solution

### PR DESCRIPTION
**Pull request by yuyi-sl**
### Description
New system-wide network capture feature.

### Motivation and Context
Update documentation to reflect the new solution.

### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/mobile-apps/features/network-capture.md [hunk 1]
```diff
@@ -126,12 +216,27 @@ Our network capture code depends on the fact that these classes are not obfuscat
...
 - [Sauce Labs Test Results](/test-results)
 - [Sauce Labs Insights](/insights)
+- [Visualize HAR Files With The New React Network Viewer Component](https://opensource.saucelabs.com/blog/react_network_viewer)
+
+### Viewing HAR Files
+
+You need a HAR viewer to read HAR files. Sauce Labs provides a [React Network Viewer](https://opensource.saucelabs.com/network-viewer/) HAR
+viewer for your convenience.
+
+Alternatively, some other commonly used HAR viewers include:
```
Comment (kristofmuhi) [inline docs/mobile-apps/features/network-capture.md:237]: Lets add here charlesproxy and fiddler. that is the most common HAR viewer for mobile folks

File: docs/mobile-apps/features/network-capture.md [hunk 2]
```diff
@@ -7,92 +7,182 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
-- Pinpointing requests slowing down the loading process
-- Locating faulty API calls
+1. On Sauce Labs, click **Live** > **Mobile App**.
+2. Upload your mobile app to Sauce Labs [through our UI](/mobile-apps/app-storage/#uploading-apps-via-ui) or [our REST API](/mobile-apps/app-storage/#uploading-apps-via-rest-api).
+3. After you’ve uploaded your app, return to the **Live** > **Mobile App** page, hover your mouse over your app, then select **Settings**.<br/><img src={useBaseUrl('img/mobile-apps/networkcapturescr.png')} alt="Mobile app settings navigation" width="600"/>
+4. Under **Default Settings**, toggle Instrumentation and Network Capture to enable the feature.<br/><img src={useBaseUrl('img/mobile-apps/networkcapturescr2.png')} alt="Mobile app settings navigation" width="800"/>
+5. For Automated Testing only: add the networkCapture capability to your test script. Click the link below corresponding to your framework:
+
+    - [Appium](/dev/test-configuration-options/#saucelabsnetworkcaptureenabled)
+    - [Espresso via saucectl](/mobile-apps/automated-testing/espresso-xcuitest/espresso/#networkcapture)
```
Comment (kristofmuhi) [inline docs/mobile-apps/features/network-capture.md:180]: lets call out this comes later

File: docs/mobile-apps/features/network-capture.md [hunk 3]
```diff
@@ -7,92 +7,182 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 You can also download a HAR file programmatically using the following API request:
 
-```curl
-curl --compressed \
--O https://{SAUCE_USERNAME}:{SAUCE_ACCESS_KEY}@{DATA_CENTER}.saucelabs.com/v1/rdc/jobs/{JOB_ID}/network.har
+<Tabs
+groupId="dc-url"
+defaultValue="us"
+values={[
+{label: 'United States', value: 'us'},
```
Comment (kristofmuhi) [inline docs/mobile-apps/features/network-capture.md:146]: do we even us-east?! hahaha

File: docs/mobile-apps/features/network-capture.md [hunk 4]
```diff
@@ -7,92 +7,182 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
+#### Filter by content type
+Filter requests by their content type by selecting the desired category from the options available.
+<img src={useBaseUrl('img/mobile-apps/network-capture-type-filter.png')} alt="Inspect network logs" width="700"/>
+
+#### Requests table
+The main table displays all captured requests with the following columns:
+* Path: The name of the request. Hover over it to view the full resource, including the domain name.
+* Domain: The domain of the request.
+* Status: The HTTP status code.
```
Comment (kristofmuhi) [inline docs/mobile-apps/features/network-capture.md:97]: so lets call everywhere HTTP/S so nobody can call it out that why we support only HTTP

File: docs/mobile-apps/features/network-capture.md [hunk 5]
```diff
@@ -7,92 +7,182 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
-4. Under **Default Settings**, toggle Instrumentation and Network Capture to enable the feature.<br/><img src={useBaseUrl('img/mobile-apps/networkcapturescr2.png')} alt="Mobile app settings navigation" width="800"/>
-5. For Automated Testing only: add the networkCapture capability to your test script. Click the link below corresponding to your framework:
+On iOS/iPadOS devices, this feature works seamlessly without requiring any [resigning](/mobile-apps/automated-testing/ipa-files/#sauce-labs-resigning-enablements)
+or changes to the app, making it simple and efficient to use.
 
-   - [Appium](/dev/test-configuration-options/#saucelabsnetworkcaptureenabled)
-   - [Espresso via saucectl](/mobile-apps/automated-testing/espresso-xcuitest/espresso/#networkcapture)
-   - [XCUITest via saucectl](/mobile-apps/automated-testing/espresso-xcuitest/xcuitest/#networkcapture)
+For Android devices, a minor modification in the manifest is necessary, but all required changes will be handled by us. This means no
```
Comment (kristofmuhi) [inline docs/mobile-apps/features/network-capture.md:40]: lets call out here that we need to modify the manifest file and call out this to ensure the SSL pinning is disabled, and that our certificate is accepted. 
additionally call out we need the debug version of their app, and make sure that in the secureSDK repackaging prevention is disabled

**Review by kristofmuhi — APPROVED** [review]

**Review by kristofmuhi — APPROVED** [review]

File: docs/mobile-apps/features/network-capture.md [hunk 6]
```diff
@@ -7,92 +7,182 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
-- Identifying browser requests that time out
-- Pinpointing requests slowing down the loading process
-- Locating faulty API calls
+1. On Sauce Labs, click **Live** > **Mobile App**.
+2. Upload your mobile app to Sauce Labs [through our UI](/mobile-apps/app-storage/#uploading-apps-via-ui) or [our REST API](/mobile-apps/app-storage/#uploading-apps-via-rest-api).
+3. After you’ve uploaded your app, return to the **Live** > **Mobile App** page, hover your mouse over your app, then select **Settings**.<br/><img src={useBaseUrl('img/mobile-apps/networkcapturescr.png')} alt="Mobile app settings navigation" width="600"/>
+4. Under **Default Settings**, toggle Instrumentation and Network Capture to enable the feature.<br/><img src={useBaseUrl('img/mobile-apps/networkcapturescr2.png')} alt="Mobile app settings navigation" width="800"/>
+5. For Automated Testing only: add the networkCapture capability to your test script. Click the link below corresponding to your framework:
+
+    - [Appium](/dev/test-configuration-options/#saucelabsnetworkcaptureenabled)
```
Comment (diemol) [inline docs/mobile-apps/features/network-capture.md:179]:
```suggestion
    - [Appium](/dev/test-configuration-options/#networkcapture)
```

Comment (diemol) [reply docs/mobile-apps/features/network-capture.md:179]: I think the link is wrong, I added a suggestion.

File: docs/mobile-apps/features/network-capture.md [hunk 7]
```diff
@@ -7,92 +7,182 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 
-- Chrome devtools builtin [HAR imports](https://developer.chrome.com/blog/new-in-devtools-62/#har-imports)
-- Google Admin Toolbox [HAR Analyzer](https://toolbox.googleapps.com/apps/har_analyzer/)
-- Software is Hard blog [HTTP Archive Viewer](http://www.softwareishard.com/har/viewer/)
-- Firefox network monitoring builtin [HAR imports](https://firefox-source-docs.mozilla.org/devtools-user/network_monitor/toolbar/index.html)
+</TabItem>
+</Tabs>
 
-## HAR Files
+## Instrumented Network Traffic Capture
```
Comment (diemol) [inline docs/mobile-apps/features/network-capture.md:169]: Do we need to clarify again that this is only for Android?

**Review by diemol — COMMENTED** [review]
Left a couple of comments.

File: docs/mobile-apps/features/network-capture.md (hunk 7 above)
Comment (kristofmuhi) [reply docs/mobile-apps/features/network-capture.md:169]: yes

**Pull request merged**